File: src/main/java/com/kamikazejam/datastore/util/Color.java
package com.kamikazejam.datastore.util;

import org.bukkit.ChatColor;
import org.jetbrains.annotations.Contract;

public class Color {
    @Contract("null -> null")
    public static String t(String msg) {
        if (msg == null) { return null; }
        return ChatColor.translateAlternateColorCodes('&', msg);
    }
}

File: src/main/kotlin/com/kamikazejam/datastore/util/Color.kt
package com.kamikazejam.datastore.util

import org.bukkit.ChatColor
import org.jetbrains.annotations.Contract

object Color {
    @Contract("null -> null")
    fun t(msg: String?): String? {
        if (msg == null) {
            return null
        }
        return ChatColor.translateAlternateColorCodes('&', msg)
    }
    fun tNotNull(msg: String): String {
        return ChatColor.translateAlternateColorCodes('&', msg)
    }
}
